### ISSUE TITLE
Staging Review finding: Unnecessary `role="search"`

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#99045]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** 10-10 Health Apps
**Product:** 10-10CG
**Feature:** Improve Facility Selection

## Findings details
**VA.gov Experience Standard - issue:** [User encounters no repetition or redundancy.](https://design.va.gov/about/experience-standards/#user-encounters-no-repetition-or-redundancy)
**VA.gov Experience Standard - category:** [Efficiency](https://design.va.gov/about/experience-standards/#category-efficiency)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

On [/locator/](https://staging.va.gov/family-and-caregiver-benefits/health-and-disability/comprehensive-assistance-for-family-caregivers/apply-form-10-10cg/veteran-information/va-medical-center/locator), the search input is inside a `va-card` component, and the `va-card` component has a `role="search"` attribute. Since `va-search-input` already has a 'role="search"` included as part of the component, the result is that there are two `search` landmarks defined, one inside the other.

This results in some unnecessary repetition, with the role of the field announced multiple times to screen reader users.

<details><summary>Link, screenshot or steps to recreate</summary>

![Facility locator page with a card containing a search input labeled "City, state, or postal code (* Required)."](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/working-with-vsp/vsp-collaboration-cycle/staging-review-images/51980-Brian-1.png?raw=true)

Coded as:
```
<va-card role="search" background="true" class="hydrated">
   <div class="">
      <label for="facility-search" class="vads-u-margin-top--0 vads-u-margin-bottom--1">City, state or postal code<span class="vads-u-color--secondary-dark"> (*Required)</span></label>
       <va-search-input label="City, state or postal code" value="" uswds="true" class="hydrated"></va-search-input>
   </div>
</va-card>
```

</details>

## Recommended action

Remove `role="search"` from the card. The search component should take care of that for you.

## References

- **Accessibility Defect Severity:** [3: Medium. Should be fixed in 1-3 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-3)
- **WCAG Success Criteria:** N/A
- **Modality:** Screen reader
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.
